components/types/User.ts

```diff
@@ -0,0 +1,10 @@
+export interface User {
```

Comment:
just to be safe: Das interface matched mit den Daten die vom Backend kommen?

Reply:
ja